soc/board_specific_workflows/lattice_crosslink_nx_evn.py

```diff
@@ -23,9 +23,57 @@
 from litespi.phy.generic import LiteSPIPHY
 from litespi import LiteSPI
 
+from migen import Signal, If, Module
+from litex.soc.interconnect.csr import CSRField
+from litex.soc.interconnect.csr import CSRStatus
+from litex.soc.interconnect.csr import CSRStorage
+from litex.soc.interconnect.csr import AutoCSR
+
 KB = 1024
 MB = 1024 * KB
 
+
+class Counter(Module, AutoCSR):
```

Comment:
It might be better to give this a more specific name like SPIFlashCounter so that the corresponding CSR function names in the C code are a bit more obvious.

Reply:
The module's name has been changed.